blocks/top-table-list-block/features/top-table-list/_children/horizontal-overline-image-story-item.jsx

```diff
@@ -127,17 +127,17 @@ const HorizontalOverlineImageStoryItem = (props) => {
                   {dateTmpl()}
                 </div>
               </div>
-          )} */}
-          {/* {videoUUID && (
+          )}
+          {/* videoUUID && (
             <Video
               uuid={videoUUID}
               autoplay={false}
               aspectRatio={0.75}
               org="arcbrands"
               env="sandbox"
             />
-          )} */}
-          {customFields.showImageLG && /*! videoUUID && */ (
+          ) */}
+          {customFields.showImageLG /* && !videoUUID */ && (
```

Comment:
commented out? videoUUId inline?

Reply:
I'd originally added video as a promo image option, but that wasn't quite working yet and getting it working was outside the scope for the ticket so I just commented it out.